### PR TITLE
Add document for MPI example

### PR DESCRIPTION
**Pull request by TommyLike**
**What this PR does / why we need it**:
Add document for MPI example.

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by TommyLike** [comment]
wait for #791

File: example/volcano/integrations/mpi/mpi-example.yaml [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+#   was unable to find any relevant network interfaces:
+#
+#   Module: OpenFabrics (openib)
+#   Host: lm-mpi-job-mpiworker-1
+#
+#   Another transport will be used instead, although this may result in
+#   lower performance.
+#   --------------------------------------------------------------------------
+#   Hello world from processor lm-mpi-job-mpiworker-1, rank 1 out of 2 processors
+#   Hello world from processor lm-mpi-job-mpiworker-0, rank 0 out of 2 processors
```
Comment (hzxuzhonghu) [inline example/volcano/integrations/mpi/mpi-example.yaml:33]: I get 

```
Hello world from processor lm-mpi-job-mpiworker-0, rank 0 out of 2 processors
Hello world from processor lm-mpi-job-mpiworker-0, rank 1 out of 2 processors

```

Comment (hzxuzhonghu) [reply example/volcano/integrations/mpi/mpi-example.yaml:33]: When -np  4, the workers processor balanced

File: example/volcano/integrations/mpi/mpi-example.yaml [hunk 2]
```diff
@@ -21,9 +59,10 @@ spec:
                 - /bin/sh
                 - -c
                 - |
-                  MPI_HOST=`cat /etc/volcano/mpiworker.host | tr "\n" ","`;
+                  sleep 10s
```
Comment (hzxuzhonghu) [inline example/volcano/integrations/mpi/mpi-example.yaml:62]: why?

File: example/volcano/integrations/mpi/mpi-example.yaml [hunk 3]
```diff
@@ -21,9 +59,10 @@ spec:
...
                   mkdir -p /var/run/sshd; /usr/sbin/sshd;
-                  mpiexec --allow-run-as-root --host ${MPI_HOST} -np 2 mpi_hello_world > /home/re;
+                  mpiexec --allow-run-as-root --hostfile /etc/volcano/mpiworker.host -np 2 mpi_hello_world;
+                  sleep 10s
```
Comment (hzxuzhonghu) [inline example/volcano/integrations/mpi/mpi-example.yaml:65]: ditto

File: example/volcano/integrations/mpi/mpi-example.yaml (hunk 2 above)
Comment (TommyLike) [reply example/volcano/integrations/mpi/mpi-example.yaml:62]: It's mentioned in NOTES section

File: example/volcano/integrations/mpi/mpi-example.yaml (hunk 1 above)
Comment (TommyLike) [reply example/volcano/integrations/mpi/mpi-example.yaml:33]: It surely will change, it's only an example

**Pull request merged**